### ISSUE TITLE
Invalid bootstrap token when adding new node

### ISSUE THREAD

**Issue by calcium90** [thkukuk/kubic-control#24]
When adding a new node to the cluster:
```
# kubicctl node add devkubewkr02
Generate new token ...
Error invoking kubeadm: exit status 1
```
I found this in the kubicd logs:
```
Jul 23 10:58:40 devkubemst01 kubicd[1740]: time="2020-07-23T10:58:40Z" level=info msg="Received: add node  devkubewkr02"
Jul 23 10:58:40 devkubemst01 kubicd[1740]: time="2020-07-23T10:58:40Z" level=info msg="Token to join nodes too old, creating new one"
Jul 23 10:58:40 devkubemst01 kubicd[1740]: time="2020-07-23T10:58:40Z" level=info msg="Executing /usr/bin/kubeadm: [kubeadm token create --print-join-command 2>/dev/null]"
Jul 23 10:58:41 devkubemst01 kubicd[1740]: time="2020-07-23T10:58:41Z" level=error msg="Error invoking kubeadm: exit status 1\nthe bootstrap token \"2>/dev/null\" was not of the form \"\\\\A([a-z0-9]{6})\\\\.([a-z0-9]{16})\\\\z\"\nTo >
Jul 23 10:58:41 devkubemst01 kubicd[1740]: time="2020-07-23T10:58:41Z" level=info msg="Function: /api.Kubeadm/AddNode, Caller: admin, Duration: 782.466674ms, Error: <nil>"
```
I also tried manually creating a token with kubeadm token create but kubicctl still tries to create a new one and fails.

Is there any way for me to manually create a token and get kubicctl to see it (so it doesn't try and generate a new one) so I can work around this?

The admin node and worker both running MicroOS 20200720, kubicctl 0.10.0-1.1.

**Comment by thkukuk** [comment]
This is already fixed (See issue #22, release 0.10.1 contains that, too) and happens, if salt-master and kubicd runs on the first master node, not if they run on a different node.

**Comment by calcium90** [comment]
That fix works for me, thanks.
